fhir-converter/src/templates/cda/ValueSet/MedicationRequestStatus.hbs

```diff
@@ -0,0 +1,14 @@
+{{! https://www.hl7.org/fhir/R4/valueset-medicationrequest-status.html }}
```

Comment:
Great that you're adding the link to the FHIR values. Is there a link to the mapping table or at least the respective C-CDA values?

Reply:
good call - added